charts/datadog/README.md

```diff
@@ -47,7 +47,11 @@ helm install --name <RELEASE_NAME> \
 ```
 
 By default, this Chart creates a Secret and puts an API key in that Secret.
-However, you can use manually created secret by setting the `datadog.apiKeyExistingSecret` value. After a few minutes, you should see hosts and metrics being reported in Datadog.
+However, you can use manually created secrets by setting the `datadog.apiKeyExistingSecret` and/or `datadog.appKeyExistingSecret` values (see [Creating a Secret](#create-and-provide-a-secret-that-contains-your-datadog-api-keys), below).
+
+**Note:** When creating the secret(s), be sure to name the key fields `api-key` and `app-key`.
+
```

Comment:
**Note:**
* The same secret can be referenced in `datadog.apiKeyExistingSecret` and `datadog.appKeyExistingSecret`.
* When creating the secret(s), be sure to name the key fields `api-key` and `app-key`.